### ISSUE TITLE
LuiMenuDropdown

### ISSUE THREAD

**Issue by rhmkstk** [LUI-UI/lui-vue#156]
### Describe the bug

Keyboard focus does not work when luimenuitems render with v-for

### Do you know how to fix the issue

None

### Are you willing to participate in fixing this issue and create a pull request with the fix

None

**Comment by rhmkstk** [comment]
tested with v-for there is no problem
